examples/plot_compare_reduction.py

```diff
@@ -72,4 +86,45 @@
 plt.ylabel('Digit classification accuracy')
 plt.ylim((0, 1))
 plt.legend(loc='upper left')
+
+###############################################################################
+# Caching transformers within a ``Pipeline``
```

Comment:
Ok let's revert back this example to its previous state and instead enable the memory on the following example that really benefit from caching:

`grid_search_text_feature_extraction.py`